docs/components/table.md

```diff
@@ -85,12 +85,107 @@ Lessen the table cell padding
   isSmall />
 ```
 
+## Events
+Bind any DOM [events](https://developer.mozilla.org/en-US/docs/Web/Events) to
+various parts of the table.
+
+### Rows
+- `@row:<event>` - returns the `Event`, the row item, and the type: `row`
+
+```vue
+<KTable @row:click="rowHandler" @row:dblclick="rowHandler">
+```
+
+### Cells
+
+- `@cell:<event>` - returns the `Event`, the cell value, and the type: `cell`
+
+```vue
+<KTable @cell:mouseout="cellHandler" @cell:mousedown="cellHandler">
+```
+
+#### Example
+
+```vue
+<template>
+  <div>
+    <div v-if="eventType">
+      {{eventType}} on: {{row}}
+    </div>
+    <div v-else>Waiting</div>
+
+    <KTable 
+      class="clickable-rows"
+      :options="tableOptions"
+      @row:click="actionRow" 
+      @cell:mouseover="actionRow"
+    />
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      row: null,
+      eventType: ''
+    }
+  },
+  methods: {
+    actionRow (e, row, type) {
```

Comment:
is this correct?
`e` = Whole DOM Event
`row` = the KTable row item
`type` = DOM Event type (click, hover, etc)

Reply:
`type` is actually `cell` or `row`. This helps if you want a master method event handler.
`e.type` will give you click hover etc.

Reply:
ah gotcha

Reply:
ah gotcha, cool beans